oceanprotocol/uploader-ui-lib
# Wrapping matic for payment

Fixes #58 


Steps for testing: 

1. Go to the preview link
2. make sure that you have Matic in your wallet but no Wrapped Matic. 
3. connect your wallet and check that you are on polygon mainnet. 
4. Choose a file to upload. 
5. Request a quote
6. Wrap your matic
7. Upload the file
8. Request the DDO link
9. Go to arweave.net/{TRANSACTION_ID} to check that the upload worked
